.drone.yml

```diff
@@ -122,6 +122,33 @@ trigger:
   event:
   - push
 
+
+---
+kind: pipeline
+type: exec
+name: integration-tests-jail-unjail
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: integration-tests
+  commands:
+  - cd integration-tests/jail
+  - ./run.sh
+- name: teardown
+  commands:
+  - cd integration-tests/jail
+  - . ./run_open_port.sh
+  - . ./run_test_env.sh
```

Comment:
why tearing down needs `run_open_port.sh`? That script writes some environment -- I guess that should already be done in `run.sh` ?

Reply:
without it, docker compose fails in shutting down, because it uses environment variables.

Reply:
But different runs of run_open_port will have different ports I guess?

Reply:
it is created just onetime per test.
choose available port using socket api.

Reply:
> But different runs of run_open_port will have different ports I guess?
> It should be ok though, docker maybe not use the actual value of these environment variables when closing, only needs them exists.

yes, that may be possible , i'll experiment.

Reply:
> But different runs of run_open_port will have different ports I guess?
> It should be ok though, docker maybe not use the actual value of these environment variables when closing, only needs them exists.

yes, that' correct.
i fixed